hello-world-attest-fn-call/config.toml

```diff
@@ -0,0 +1,18 @@
+# a set of acceptable server enclave measurements for CLI to interact with
+acceptable_measurements = [
+    # platform and code pair identifies the acceptable server implementation
+    { platform = "nitro", code = "edd5bfcd857e20ff5f151fbb8160f54f638ec5bddee5922d2c5db3575c9c4db14b88a53d90917965c705bbdba48ee657.4b4d5b3661b3efc12920900c80e126e4ce783c522de6c02a2a5bf7af3a2b9327b86776f188e4be1c1c404a129dbda493.dbecae9b2e9e79b10bcf55138a7d028e565d85cea7b095639000304e1d42d6d2d6839ebeea03d55e14c03ec474885ae3" },
+    # a measurement that is valid for when the server is running on a non-TEE environment
+    { platform = "plain", code = "plain" },
+    # a measurement that is valid for when the server is running in debug mode on nitro
+    { platform = "nitro", code = "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000" },
+]
+# token authorizing CLI's access to the server
+auth_token = "eyJvcmciOiI2NjRmOTgxOTU3ZjhkODAwMDExZDU2NTUiLCJpZCI6IjU0MDhhOGUwNDY0NTQ0NWU5ZGE3ZGYwOTQ5Y2FjMDU2IiwiaCI6Im11cm11cjEyOCJ9"
+
+# the server that the CLI interacts with.
+# `local-server` is a special host value that will start a local attestation
```

Comment:
consider removing all this extra config, it adds more complexity than is needed  for this demo

Reply:
leaving since we're targetting the local server